site/src/analytics-active-installations.ts

```diff
@@ -25,13 +24,20 @@ export class AnalyticsActiveInstallations extends LitElement {
 
   @query("google-chart") private _chart?: GoogleChart;
 
-  protected firstUpdated(_changedProperties: PropertyValues) {
-    super.firstUpdated(_changedProperties);
+  public connectedCallback(): void {
+    super.connectedCallback();
     window.addEventListener("resize", () => {
       this._chart?.redraw();
     });
   }
 
+  public disconnectCallback(): void {
+    super.disconnectCallback();
+    window.removeEventListener("resize", () => {
+      this._chart?.redraw();
+    });
+  }
+
```

Comment:
this will not work. You are defining a new function here, so that reference won't match the old listener. Instead, store the resize listener on the class instance.
```suggestion
    window.addEventListener("resize", this._handleResize);
  }

  public disconnectCallback(): void {
    super.disconnectCallback();
    window.removeEventListener("resize", this._handleResize);
  }

  private _handleResize = () => {
    this._chart?.redraw();
  }
```